test/evergreen.yml

```diff
@@ -5523,7 +5508,6 @@ buildvariants:
     - name: make-check-test
     - name: fops
     - name: time-shift-sensitivity-test
-    - name: linux-directio
       distros: rhel80-build
```

Comment:
`distros` should also go here and on line 5287 (Tod pointed this out, I'm just expanding it to a few more places.)